core/webgl-compatibility/src/Capabilities.ts

```diff
@@ -76,6 +76,7 @@ const buggyIntelMatchers = [
 // Regexes to match Mali GPUs known to suffer from GraphicsDriverBugs.msaaWillHang.
 const buggyMaliMatchers = [
   /Mali-G71/,
+  /Mali-G72/,
```

Comment:
msaaWillHang is a separate issue from the transparency 'allowFloatRender' workaround stuff done below.
`buggyMaliMatchers` probably was not the best original name.
I think it's arguable that it's very likely that Malis in general will suffer from the MSAA hang bug too.

Reply:
Was a hang observed for MSAA with this chip too?  Or just transparency issues?  Just curious.  Even if not, I suppose you're right that it's probably safe to just apply it.

Reply:
Confirmed hang.